projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -765,6 +767,8 @@ export class IgxGridCRUDService extends IgxRowAddCrudState {
             this.grid.navigateTo(rowIndex, visibleColIndex);
         }
 
+        this.grid.notifyChanges(false);
+
```

Comment:
From what I can see, this is also not required anymore (if it was indeed). Unless you want to refactor all other calls into this method that end up calling `notifyChanges` as well one way or another, I'd rather not have this extra here.